e2e/helpers/actions/giveModelAccess.ts

```diff
@@ -0,0 +1,24 @@
+import type { User } from "../auth";
+import type { Model } from "../objects";
+import { ModelPermission } from "../objects";
+
+import { GiveAccess } from "./utils/give-access";
+
+const jimmAccess = new Map([
+  [ModelPermission.READ, "reader"],
+  [ModelPermission.WRITE, "writer"],
+  [ModelPermission.ADMIN, "administrator"],
+]);
+
+/**
+ * Give a user access to a model.
+ */
+export class GiveModelAccess extends GiveAccess<Model> {
```

Comment:
I've renamed this to `Give` instead of `Grant` as this uses a different method of giving access for JIMM.